lib/rubocop/cop/lint/void.rb

```diff
@@ -5,6 +5,10 @@ module Cop
     module Lint
       # This cop checks for operators, variables, literals, and nonmutating
       # methods used in void context.
+      # It correctly works for methods inherited from Enumerable. Does not work
+      # for others, except for `#defined?` method.
+      # This cop can yield some false positives in case of Enumerable-like
+      # methods.
```

Comment:
@bbatsov Sorry it took so long but I rewrote this part too many times and I still don't feel that it's right